packages/host/app/components/operator-mode/host-submode.gts

```diff
@@ -15,53 +27,200 @@ interface HostSubmodeSignature {
 }
 
 export default class HostSubmode extends Component<HostSubmodeSignature> {
+  @consume(GetCardContextName) private declare getCard: getCard;
```

Comment:
Don't do this. Use a service.

Reply:
ooof this was what you feared @lukemelia. perhaps there is a good way to teach this to the rest of the team?

Reply:
I'll create one

Reply:
Likely should just do `import { getCard } from '@cardstack/host/resources/card-resource';`